types/screener.d.ts

```diff
@@ -0,0 +1,66 @@
+//
+// Typings for `screener-runner`
+//
+
+type ScreenerRunnerKeys = {
+  alt: string
+  control: string
+  enter: string
+  escape: string
+  return: string
+  shift: string
+  tab: string
+  leftArrow: string
+  upArrow: string
+  rightArrow: string
+  downArrow: string
+}
+
+type ScreenerRunnerSteps = {
+  /** This will click on the first element matching the provided css selector. */
+  click(selector: string): ScreenerRunnerSteps
+
+  /** This will capture a visual snapshot. */
+  snapshot(name: string): ScreenerRunnerSteps
+
+  /** this will move the mouse over the first element matching the provided css selector. */
+  hover(selector: string): ScreenerRunnerSteps
+
+  /** This will press and hold the mouse button over the first element matching the provided css selector. */
+  mouseDown(selector: string): ScreenerRunnerSteps
+
+  /** This will release the mouse button. selector is optional. */
+  mouseUp(selector: string): ScreenerRunnerSteps
+
+  /** This will set cursor focus on the first element matching the provided css selector. */
+  focus(selector: string): ScreenerRunnerSteps
+
+  /** This will set the value of the input field matching the provided css selector. */
+  setValue(selector: string, value: string): ScreenerRunnerSteps
```

Comment:
has an optional parameter as well:
```suggestion
  setValue(selector: string, value: string, options?: any): ScreenerRunnerSteps
```